### PR TITLE
Update related to ADFS #3151 as important note.

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added important note #3151 

Copy review @nenonix @JohanFreelancer9 is required on this one guys.
Thank you in advance!

### CODE REVIEW

File: Teams/room-systems/with-exchange-online.md [hunk 1]
```diff
@@ -86,7 +86,10 @@ Import-PSSession $Session -DisableNameChecking
    > [!NOTE]
    > [Azure Active Directory PowerShell 2.0](https://docs.microsoft.com/powershell/azure/active-directory/overview?view=azureadps-2.0) is not supported. 
 
-  ``` PowerShell
+   > [!IMPORTANT]
+   > In case that your tenant has ADFS Implemented and a federated user, please proceed to convert the user to managed while you do this set up and then return it back to federated.
```
Comment (JohanFreelancer9) [inline Teams/room-systems/with-exchange-online.md]:
```suggestion
   > If your tenant is a federated user and has ADFS Implemented, the user should be converted to a managed user while you are doing this setup, and then be return to federated on completion.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Pinged PM.

**Comment by LanaChin** [comment]
Thanks j0rt3g4. The folder structure of this topic has changed so I opened a separate PR for this change. The topic is now updated. Closing this without merging.

**Pull request closed without merging**